Code/Framework/AzToolsFramework/Tests/Prefab/PrefabLoadTemplateTests.cpp

```diff
@@ -330,7 +346,9 @@ namespace UnitTest
     TEST_F(PrefabLoadTemplateTest, LoadTemplate_LoadFromString_CorruptedReturnsInvalidTemplateId)
     {
         AZStd::string corruptPrefab = "{ Corrupted PrefabDom";
+        AZ_TEST_START_TRACE_SUPPRESSION;
         TemplateId templateId = m_prefabLoaderInterface->LoadTemplateFromString(corruptPrefab);
+        AZ_TEST_STOP_TRACE_SUPPRESSION_NO_COUNT;
```

Comment:
Just curious, what's the reason behind adding 'no_count' in some places vs specifying exact count in other places?

Reply:
there were multiple errors being triggered, when is 1-2 I put the count

Reply:
I think it's better to add the actual count so that if more than expected errors are reported in the future, they would be caught by these tests instead of suppressing them.

Thanks for adding this support to AzToolsFramework btw. This is awesome!

Reply:
Its better, but requires a lot of debugging to count the amount of errors it hits in each case. Considering the original write was not considering errors at all, at least is a step forward.
Will try to add the counts if I have some more time.

Reply:
Found a way to do this easily, so will change all those checks